### DISCUSSION TITLE
Welcome to phoebe 1 (legacy) discussions!

### DISCUSSION THREAD

**Discussion by aprsa** [phoebe-project/phoebe1#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Alakay2** [comment]
Hi Everyone,
I am using the mcmc script (available at https://sourceforge.net/p/phoebe/mailman/message/33650955/) for determining the parameter errors in the determined parameters of an eclipsing binary using Phoebe-1. 
I ran that script with all the input files and after some outputs on terminal got something like this:

=========================================
LCs: 1; RVs: 2; LD model: Logarithmic law
LCs: 1; RVs: 2; LD model: Logarithmic law
LCs: 1; RVs: 2; LD model: Logarithmic law
==========================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   PID 114130 RUNNING AT fedora
=   EXIT CODE: 9
=   CLEANING UP REMAINING PROCESSES
=   YOU CAN IGNORE THE BELOW CLEANUP MESSAGES
==========================================
YOUR APPLICATION TERMINATED WITH THE EXIT STRING: Segmentation fault (signal 11)
This typically refers to a problem with your application.
Please see the FAQ page for debugging suggestions
#######
I thought this was due to some system related issues (may be I need more space to run the program). I also tried to decrease the number of iterations in the program but after showing some outputs on terminal :
#######
.....
PHOEBE-lib warning: parameter lookup failed for qualifier gui_rvplot_synthetic.
PHOEBE-lib warning: parameter lookup failed for qualifier gui_rvplot_synthetic.
PHOEBE-lib warning: parameter lookup failed for qualifier gui_lc_plot_x_offset.
PHOEBE-lib warning: parameter lookup failed for qualifier gui_rv_plot_observed.
PHOEBE-lib warning: parameter lookup failed for qualifier gui_rv_plot_fine.
PHOEBE-lib warning: parameter lookup failed for qualifier gui_rv_plot_zoom.
PHOEBE-lib warning: parameter lookup failed for qualifier gui_rv_plot_residuals.
PHOEBE-lib warning: parameter lookup failed for qualifier gui_rv_plot_coarse.
PHOEBE-lib warning: parameter lookup failed for qualifier gui_rv_plot_phstart.
PHOEBE-lib warning: parameter lookup failed for qualifier gui_rv_plot_y_offset.
PHOEBE-lib warning: parameter lookup failed for qualifier gui_lcplot_synthetic.
PHOEBE-lib warning: parameter lookup failed for qualifier gui_ld_model_autoupdate.
PHOEBE-lib warning: parameter lookup failed for qualifier gui_lc_plot_x.
PHOEBE-lib warning: parameter lookup failed for qualifier gui_lc_plot_y.
#######
It stops. Can you suggest something to me about it?

Thanks,
Alax
